vaadin-platform-test/pom-dev.xml

```diff
@@ -0,0 +1,235 @@
+                                    <goal>integration-test</goal>
+                                    <goal>verify</goal>
+                                </goals>
+                                <configuration>
+                                    <includes>
+                                        <include>DefaultDevBundleIT,ChromeComponentsIT</include>
+                                    </includes>
+                                </configuration>
+                            </execution>
+                        </executions>
+                        <configuration>
+                            <forkCount>2</forkCount>
+                        </configuration>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.eclipse.jetty</groupId>
+                        <artifactId>jetty-maven-plugin</artifactId>
+                        <version>${jetty.version}</version>
+                        <configuration>
+                            <stopPort>9999</stopPort>
+                            <stopWait>5</stopWait>
+                            <stopKey>foo</stopKey>
+                            <systemProperties>
+                                <vaadin.ce.dataDir>${project.build.directory}</vaadin.ce.dataDir>
+                                <vaadin.productionMode>false</vaadin.productionMode>
+                            </systemProperties>
+                        </configuration>
+                        <executions>
+                            <execution>
+                                <id>start-jetty</id>
+                                <phase>pre-integration-test</phase>
+                                <goals>
+                                    <goal>start</goal>
+                                </goals>
+                            </execution>
+                            <execution>
+                                <id>stop-jetty</id>
+                                <phase>post-integration-test</phase>
+                                <goals>
+                                    <goal>stop</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <artifactId>maven-antrun-plugin</artifactId>
+                        <version>3.0.0</version>
+                        <executions>
+                          <execution>
+                            <phase>validate</phase>
+                            <goals>
+                                <goal>run</goal>
+                            </goals>
+                            <configuration>
+                              <target>
+                                <!-- Do not read @Theme annotations from the bundle application class -->
+                                <delete dir="../vaadin-dev-bundle/target/classes/com/vaadin/devbundle" includeemptydirs="true"></delete>
```

Comment:
This looks super wrong. Why don't you delete the class in the `vaadin-dev-bundle` project instead?

Reply:
I tried in the bundle maven install phase, but then maven regenerates the class again in the compile time of the vaadin-platform-test module because it depends on the other.

Reply:
Changing the maven phase it seems working, let's see TC